server/gitrest/packages/gitrest-base/src/utils/definitions.ts

```diff
@@ -5,6 +5,7 @@
 
 import fsPromises from "fs/promises";
 import * as git from "@fluidframework/gitresources";
+import type { MakeDirectoryOptions } from "fs";
```

Comment:
nit: not sure if this is why build is failing or not, but this should be above the gitresources import. "Correct" import order is defined by the linter: [Biome](https://biomejs.dev/analyzer/import-sorting/#how-imports-are-sorted), [ESLint](https://eslint.org/docs/latest/rules/sort-imports)

Reply:
I noticed all recent gitrest build start to fail in the pipeline since some time yesterday. Not sure if this would fail the build but will re-order the import here :)